server/grpc_service.go

```diff
@@ -1487,12 +1520,17 @@ func (s *GrpcServer) SyncMaxTS(_ context.Context, request *pdpb.SyncMaxTSRequest
 	tsoAllocatorManager := s.GetTSOAllocatorManager()
 	// There is no dc-location found in this server, return err.
 	if tsoAllocatorManager.GetClusterDCLocationsNumber() == 0 {
-		return nil, status.Errorf(codes.Unknown, "empty cluster dc-location found, checker may not work properly")
+		return &pdpb.SyncMaxTSResponse{
```

Comment:
This gRPC method is used by the PD internally. I think you should also modify the caller to check the header error.